openbao.yaml

```diff
@@ -0,0 +1,64 @@
+package:
+  name: openbao
+  version: 2.0.2
+  epoch: 0
+  description: OpenBao exists to provide a software solution to manage, store, and distribute sensitive data including secrets, certificates, and keys.
+  copyright:
+    - license: MPL-2.0
+
+environment:
+  environment:
+    CGO_ENABLED: "0"
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - nodejs
+      - npm
+      - yarn
```

Comment:
We should be able to drop the environment here. Since we aren't building the UI, `go/build` includes any dependencies we need at buildtime. We also shouldn't need to explicitly disable CGO. We could leave `nodejs`, `npm`, and `yarn` here commented out with a note that they're to be used when the UI is available

Reply:
Good point. Refactored commenting out of UI - adding a note at the top and then same comment in all the places that need uncommenting to build the UI in the future.

Re CGO - this is how it's also built upstream:
https://github.com/openbao/openbao/blob/main/.goreleaser-template.yaml#L31
https://github.com/openbao/openbao/blob/main/Makefile#L18

Reply:
After giving it some thought I noticed that not disabling CGO may be a better choice. OpenBao seems to work without it and is built in both modes based on Makefile.